Getting-Started/Code/Debugging/Logging/index-v8.md

```diff
@@ -0,0 +1,149 @@
+  "ThreadId": 1,
+  "AppDomainId": 2,
+  "AppDomainAppId": "LMW3SVC2ROOT",
+  "MachineName": "DELLBOOK",
+  "Log4NetLevel": "INFO ",
+  "HttpRequestNumber": 1,
+  "HttpRequestId": "557f45ba-0888-4216-8723-e226d795a2f7"
+}
+```
+
+To learn more about structured logging and message templates you can read more about it over on the https://messagetemplates.org website
+
+## Writing to the log
+...
+Bad usages & why
+Good usages
+*******WARREN TODO*******
+
+## Log Levels
+Serilog uses levels as the primary means for assigning importance to log events. The levels in increasing order of importance are:
+
+1. **Verbose** - tracing information and debugging minutiae; generally only switched on in unusual situations
+1. **Debug** - internal control flow and diagnostic state dumps to facilitate pinpointing of recognised problems
+1. **Information** - events of interest or that have relevance to outside observers; the default enabled minimum logging level
+1. **Warning** - indicators of possible issues or service/functionality degradation
+1. **Error** - indicating a failure within the application or connected system
+1. **Fatal** - critical errors causing complete failure of the application
+
+The default log levels we ship with in Umbraco v8.0+ are:
+* .txt file **Debug**
+* .json file **Verbose**
+
+## Configuration
+Serilog can be configured and extended by using the two XML configuration files on disk found at `/config/serilog.config` which is used to modify the main Umbraco logging piepleine and a second configuration file that is at `/config/serilog.user.config` which is a sublogger and allows you to make modifications without effecting the main Umbraco logger.
+
+### Changing the log level
+This can be done by adding the following into either `serilog.config` or the sub logger configuration file `serilog.user.config`
+
+```xml
+<add key="serilog:minimum-level" value="Verbose" />
+```
+
+:::warning
+If you change the main Umbraco logger in serilog.config to log only **Warning** you would not be able to have the serilog.user.config sub logger to be set to **Debug** as only Warning messages and higher will flow down into the child sub logger
+:::
+
+
+### Changing the log level for specific namespaces
+This can be done by adding the following into either `serilog.config` or the sub logger configuration file `serilog.user.config`
+
+```xml
+<add key="serilog:minimum-level:override:Microsoft" value="Warning" />
+<add key="serilog:minimum-level:override:Microsoft.AspNetCore.Mvc" value="Error" />
+<add key="serilog:minimum-level:override:YourNameSpace" value="Information" />
+```
+
+:::warning
+If you change the **serilog:minimum-level** to be **Error** then the following example above would only log messages from *Microsoft.AspNetCore.Mvc* and not any warning, info or debug messages from the *Microsoft* namespace
+:::
+
+### Changing the location of the main Umbraco Log file output
+*******WARREN TODO*******
+
+### Writing to a different source
```

Comment:
I think writing to a source is a little confusing. How about "Writing log events to different storage types" or something along those lines?